tools/run_tests/run_interop_tests.py

```diff
@@ -699,25 +699,37 @@ def bash_cmdline(cmdline):
     return ['bash', '-c', ' '.join(cmdline)]
 
 
-def auth_options(language, test_case):
+def compute_engine_creds_required(language, test_case):
+    """Returns True if given test requires access to compute engine creds."""
+    language = str(language)
+    if test_case == 'compute_engine_creds':
```

Comment:
This function can be simplified to  `return test_case == 'compute_engine_creds' or test_case == 'oauth2_auth_token' and str(language) == 'c++'` and I think it remains just as readable.

Reply:
IMHO it is more readable the way it is and the 'oauth2_auth_token' with c++ is kind of an edge case so I like to call it out explicitly. If you insist, I can fix in a followup PR.